### PR TITLE
add support to downsize to 16px

### LINKED ISSUE #3493: Update icon build step to support downsampling to 16x16
Currently our build step in `@carbon/icons` only downsamples to 20/24, we should also target 16 assets as well if no bespoke version exists.

Current const for `SCALED_SIZES`: https://github.com/carbon-design-system/carbon/blob/master/packages/icons/src/build.js#L20

### PR DESCRIPTION
**Pull request by lovemecomputer**
Closes #3493 

**changes**
- adds 16px as a supported downsize target (from 32px source svg, if no 16px asset exists)

### CODE REVIEW

**Review by joshblack — APPROVED** [review]

**Comment by joshblack** [comment]
If we keep hitting this bug in `icons-angular` let me know, I think we might have to revisit that given that we're at an insane level of icons lol (800 * 4 sizes === 3200 😬)

**Comment by lovemecomputer** [comment]
@joshblack do you have any ideas on how i should proceed on resolving these build errors? the branch builds fine locally, so i'm not sure if it's a circleci thing?

**Comment by joshblack** [comment]
@lovemecomputer I think it's something related to `icons-angular` running out of memory due to the new volume of icons. Would you mind disabling it as part of this PR, alongside making an issue to look into this issue and link the PR? Will tag the angular team to take a look at it then 👍

**Comment by emyarod** [comment]
@lovemecomputer @joshblack I just opened https://github.com/carbon-design-system/carbon/issues/3519 to document the error, it looks like it's failing for all of our PRs right now

**Comment by lovemecomputer** [comment]
@emyarod thanks!! i'm actually getting a similar, but slightly different error when building this branch locally. can post details in your issue

**Comment by lovemecomputer** [comment]
marking this as ready for review, will bring up the build issue in our dev team meeting today : )

File: packages/icons-angular/tasks/build.js [hunk 1]
```diff
@@ -5,9 +5,12 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-const build = require('../src/build');
-const { reporter } = require('@carbon/cli-reporter');
+/**
+ * icons-angular disabled until memory issue resolved (circleci failing)
+ */
+// const build = require('../src/build');
```
Comment (asudoh) [inline packages/icons-angular/tasks/build.js:12]: @lovemecomputer Are you removing these commented-out code, or is it for just testing CI?

**Review by asudoh — COMMENTED** [review]
Nice change!

File: packages/icons-angular/tasks/build.js (hunk 1 above)
Comment (lovemecomputer) [reply packages/icons-angular/tasks/build.js:12]: @asudoh working off of @joshblack 's suggestion, the intent is for this to be commented out in this PR, as the memory issues cause circleci to fail. after verifying that this is a fix (seeing all tests pass on this PR), i will create a new issue to fix the memory issue in `icons-angular` and link to this PR as reference :)

Comment (asudoh) [reply packages/icons-angular/tasks/build.js:12]: OK thanks @lovemecomputer for explaining! For now, it'd be great if the commented out code here refers to the issue you create.

Comment (lovemecomputer) [reply packages/icons-angular/tasks/build.js:12]: @asudoh okay, will do!

Comment (lovemecomputer) [reply packages/icons-angular/tasks/build.js:12]: @asudoh link put in comment!

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @lovemecomputer!

**Pull request merged**